assets/sass/pages/_occupancy-index.scss

```diff
@@ -0,0 +1,5 @@
+.occupancy--index {
+  .govuk-button {
+    margin-right: 12px;
+  }
+}
```

Comment:
not 100% across the saas standards against this project - and not my strongest suit if I'm honest. Saw other examples of pages that needed custom styling so followed those approaches. Let me know if there is a better way to achieve this :)

Reply:
This is incredibly helpful - thanks!